src/scry/symbol.cr

```diff
@@ -120,28 +120,31 @@ module Scry
   end
 
   class WorkspaceSymbolProcessor
-    @@crystal_path_symbols = [] of SymbolInformation
+    @@crystal_path_symbols : Array(SymbolInformation)?
     @query_regex : Regex
 
     def initialize(@msg_id : Int32 | String, @root_path : String, @query : String)
       @workspace_files = Dir.glob(File.join(root_path, "**", "*.cr"))
-      if @@crystal_path_symbols.empty?
-        # Memoize crystal stdlib
-        @@crystal_path_symbols = search_symbols(Dir.glob(File.join(Scry.default_crystal_path, "**", "*.cr")), Regex.new(".*"))
-      end
       @query_regex = Regex.new(@query)
     end
 
     def run
       symbols = [] of SymbolInformation
       unless @query.empty?
-        symbols.concat search_symbols(@workspace_files, @query_regex)
-        symbols.concat @@crystal_path_symbols.select(&.name.match(@query_regex))
+        symbols.concat self.class.search_symbols(@workspace_files, @query_regex)
```

Comment:
Weird I thought it was simpler to call..

Another way is `WorkspaceSymbolProcessor.search_symbols` or `{{@type}}.search_symbols` (as you like, `self.class` is fine too)

Reply:
I think I prefer `self.class` :sweat_smile:

Reply:
no prob'!